developing/languages/GO.md

```diff
@@ -4,9 +4,184 @@
 
 Go aka Golang is a programming language designed by folks at Google.
 
-## Resources
+This guide provides some general resources for working in Go.
+[Web Application in Go](../../web/server/go.md)
+provides some specifics to web development.
+
+## Learning Resources
+
+### References
 
 * [The Go Programming Language Official Website](https://golang.org/)
+* [Effective Go](https://golang.org/doc/effective_go.html) (how to do things “the Go way”)
 * [Go Pointer Primer](https://github.com/trussworks/go-pointer-primer)
+* [GoDoc](https://godoc.org/) (where you can read the docs for nearly any Go package)
+* [Go wiki](https://github.com/golang/go/wiki/Learn)
+* _Book_: [The Go Programming Language](http://www.gopl.io/)
+* Advanced Testing with Go
+  [Video](https://www.youtube.com/watch?v=yszygk1cpEc)
+  and [Article](https://about.sourcegraph.com/go/advanced-testing-in-go) (great overview of useful techniques, useful for all Go programmers)
+
```

Comment:
Suggested add: Go Proverbs - https://go-proverbs.github.io/

Reply:
Also suggestion to align happy-path to the left... https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88 (Maybe this isn't the right section for this?!?)

Ooooh! Also! No magic package globals! https://peter.bourgon.org/blog/2017/06/09/theory-of-modern-go.html